pkg/agent/util/syscall/syscall_windows.go

```diff
@@ -133,6 +133,12 @@ func (a *RawSockAddrInet) String() string {
 	return a.IP().String()
 }
 
+func (a *RawSockAddrInet) deepCopy() RawSockAddrInet {
+	sockData := [26]byte{}
+	copy(a.data[:], sockData[:])
+	return RawSockAddrInet{Family: a.Family, data: sockData}
```

Comment:
`return *a` would make a copy given that `data` is an array, not a slice
am I missing something @tnqn ?

Reply:
You are right. The whole function is what struct copy does exactly.